app/src/hooks/useTeamManage.ts

```diff
@@ -0,0 +1,66 @@
+import { useMutation, useQuery, useQueryCache } from 'react-query';
+
+import { QUERY_TEAM_KEY, QUERY_TEAM_MEMBERS } from '../const/query.const';
+import { useAxios } from './useAxios';
+import { UserProfile } from './useUserProfile';
+
+export interface TeamProfile {
+  id: number;
+  name: string;
+  description: string;
+  owner_id: number;
+  updated_at: string;
+  created_at: string;
+}
+
+interface TeamExit {
+  team_id: number;
+  user_id: number;
+}
+
+export type TeamMembersResponse = UserProfile[];
+
+export const useTeamProfileQuery = () => {
+  const axios = useAxios();
+
+  return useQuery<TeamProfile>(QUERY_TEAM_KEY, async () => {
+    const response = await axios.get<TeamProfile>(QUERY_TEAM_KEY);
+    return response.data;
+  });
+};
+
+export const useShowMembersOfTeamQuery = (
```

Comment:
Przyjmijmy zasadę, że jeden hook to jeden plik. Wyciągnij pozostałe hooki do osobnych plików. Dodatkowo zmieniłbym mu nazwę na np. `useTeamMembersQuery` - show jest trochę mylące, sam hook niczego nie pokazuje